common/djangoapps/student/views.py

```diff
@@ -874,20 +849,6 @@ def change_enrollment(request, check_access=True):
 
         # Otherwise, there is only one mode available (the default)
         return HttpResponse()
-
-    elif action == "add_to_cart":
```

Comment:
@rlucioni EDIT: What I said before didn't really make sense. But yeah, the functionality is available in the auth completion view instead. It says right in the comment that "this elif clause is only used in the "auto-add after user reg/login" case, i.e. it's always wrapped in try_change_enrollment.", and I removed `try_change_enrollment`.

Can you give me a link to an example on staging or instructions for reproducing the issue?